servicecontrol/troubleshooting.md

```diff
@@ -359,3 +359,25 @@ Folder `C:\DBRecoverFolder` will now contain RavenDB recovery files. Rename the
 9. Select all of the recovery files to import them
 
 Contact [Particular support](https://particular.net/support) for assistance.
+
+
+## Low on storage space
+
+ServiceControl will halt ingestion when low on storage. This can happen if incorrect or no [capacity planning](/servicecontrol/capacity-and-planning.md) was done or that the system has grown and capacity planning wasn't re-evaluated.
+
+To mitigate growth or not having enough storage:
+
+a. Mount a new disk that is larger, stop instance, [move database](/servicecontrol/configure-ravendb-location.md), adjust drive letter or update location in configuration, start instance
+b. Lower retention, optionally compact database:
+  - [ServiceControl - Error instance setting `ServiceControl/ErrorRetentionPeriod`](/servicecontrol/creating-config-file.md#data-retention-servicecontrolerrorretentionperiod)
+  - [ServiceControl - Error instance setting `ServiceControl/EventRetentionPeriod`](/servicecontrol/creating-config-file.md#data-retention-servicecontroleventretentionperiod)
+  - [ServiceControl - Audit instance setting `ServiceControl.Audit/AuditRetentionPeriod`](/servicecontrol/audit-instances/creating-config-file.md#data-retention-servicecontrol-auditauditretentionperiod)
+  - [ServiceControl - How to compact database](/servicecontrol/db-compaction.md)
+  - [ServiceControl - How to purge expired data](/servicecontrol/how-purge-expired-data.md)
+c. Disable auditing on selecting endpoints if not all endpoint need auditing
+  - [NServiceBus - Message auditing](/nservicebus/operations/auditing.md#configuring-auditing)
+d. Filter which messages will be send to the audit queue using [NServiceBus.AuditFilter](https://github.com/NServiceBusExtensions/NServiceBus.AuditFilter)
+e. Setup multiple audit instance with different retension periods if retension requirements can vary between endpoints
+  - [ServiceControl remote instances  Sharding audit messages with split audit queues](/servicecontrol/servicecontrol-instances/remotes.md#overview-sharding-audit-messages-with-split-audit-queues)
+f. Scale-out audit storage over multiple disks and/or machines
+  - [ServiceControl remote instances  Sharding audit messages with split audit queues](/servicecontrol/servicecontrol-instances/remotes.md#overview-sharding-audit-messages-with-split-audit-queues)
```

Comment:
@ramonsmits have you checked how DocsEngine renders this? The markdown rendering isn't at all good:

![image](https://github.com/Particular/docs.particular.net/assets/677704/bfde29a2-4cf8-4053-a2c0-33eeb07277c5)